src/display.js

```diff
@@ -74,9 +75,9 @@ const display = (list) => {
     const checkBox = document.createElement('input');
     checkBox.type = 'checkbox';
     checkBox.id = element.index;
-    checkBox.onchange = function status() {
-      element.completed = this.checked;
-    };
+    checkBox.addEventListener('change', () => {
+      checkStatus(element, checkBox);
+    });
```

Comment:
- [ ] Well done for adding the change even to the checkbox. One step still remains though. Remember every action we perform in the User Interface should affect the localStorage which is our data source. Therefore, marking a task as complete in the UI must reflect the same truth in the localStorage. The screenshot below illustrates that this is currently not happening:

  ![image](https://user-images.githubusercontent.com/45002654/149401231-6f5b4a46-7821-4b1e-b35e-df63f6c172cf.png)
  
  Take a look at the local storage. Even though tasks are marked as complete, the same information is not evident in the localStorage. The completed status is still false.
  
  ## Solution:
  Adding logic in this method to update tasks in the localStorage that are being marked as complete in the UI.
  
  Please refactor as necessary to achieve this desired result.